docs/development/examples/nearlib/introduction.md

```diff
@@ -996,7 +1010,7 @@ At least one key will look like this with `FunctionCall` permissions (your publi
     "nonce": 0,                               // nonce is an incrementing counter to avoid duplicate transactions
     "permission": {
       "FunctionCall": {                       // this is a FunctionCall access key
-        "allowance": "100000000",             // with about 100 million units of gas (1 one-millionth of a NEAR)
+        "allowance": "100000000000000",       // with about 100 million units of gas (1 one-millionth of a NEAR)
```

Comment:
this is now  100 **trillion** (`100 000 000 000 000`) and 1 **one-trillionth** of a NEAR

Reply:
+1 to the first point

 Now that I'm looking at it, isn't the example actually 1 * (10^-10) of a NEAR?

Reply:
Either way, this is pretty confusing for developers (confusing myself right now). Need to think of a good way to simplify it.